### PR TITLE
DEVEM-534-update-ci [Update] ci build

### PR DESCRIPTION
**Pull request by codingBenVdS**
Update ci build 
* Build when branch != 'master'

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -27,11 +25,12 @@ jobs:
...
           gpg-private-key: ${{ secrets.MAVEN_CENTRAL_GPG_KEY }} # Value of the GPG private key to import
           gpg-passphrase: MAVEN_GPG_PASSPHRASE # env variable for GPG private key passphrase
-      - name: Publish alfresco-value-assistance-repo packages
-        if: ${{ startsWith(github.ref, 'refs/heads/master') }}
-        run: mvn --batch-mode deploy
+      #build all artifacts.
+      - name: Build alfresco-value-assistance packages
+        if: ${{ !startsWith(github.ref, 'refs/heads/master') }}
+        run: mvn clean install
         working-directory: alfresco-value-assistance-repo
```
Comment (WimCrols) [inline .github/workflows/ci.yml]: Is alfresco-value-assistance-share built too? Should `working-directory` not be removed?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -27,11 +25,12 @@ jobs:
           server-password: OSSRH_TOKEN # env variable for token in deploy
           gpg-private-key: ${{ secrets.MAVEN_CENTRAL_GPG_KEY }} # Value of the GPG private key to import
           gpg-passphrase: MAVEN_GPG_PASSPHRASE # env variable for GPG private key passphrase
-      - name: Publish alfresco-value-assistance-repo packages
```
Comment (WimCrols) [inline .github/workflows/ci.yml:30]: I missed this last time but the name on L19 is incorrect: `Set up Maven Central Repository` I don't think you are setting op a repository :)
 `Set up JDK and Maven options` for example is more correct.

**Review by WimCrols — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (codingBenVdS) [reply .github/workflows/ci.yml]: correct, it should be removed

File: .github/workflows/ci.yml (hunk 2 above)
Comment (codingBenVdS) [reply .github/workflows/ci.yml:30]: *setting the maven settings.xml file

**Review by WimCrols — APPROVED** [review]
:shipit:

**Pull request merged**